conf/application.conf

```diff
@@ -110,7 +97,7 @@ api {
   gateway.context = "organisations/interest-restriction"
 }
 
-httpHeadersWhitelist = [ "Accept", "Gov-Test-Scenario", "Content-Type", "Location", "X-Request-Timestamp", "X-Session-Id" ]
```

Comment:
any implicatations with this?

Reply:
From the READMEs of the bootstrap libraries both this and the change to play.crypto.secret look to be simply renaming the settings.